nevergrad/functions/irrigation/irrigation.py

```diff
@@ -71,10 +73,8 @@ def __init__(self, symmetry: int, n_iterations: int = 1000) -> None:
         logging.info(f"we work on {self.cropname} with variety {self.cropvariety} in {self.address}.")
 
     def set_data(self, symmetry: int, k: int):
-        self.cropname = "rice"
-        self.cropvariety = np.random.RandomState(symmetry + 3 * k + 2).choice(
-            list(self.cropd.get_crops_varieties()[self.cropname])
-        )
+        self.cropname = "betterave"
+        self.cropvariety = "sugar-beet-601"
```

Comment:
@Ethel2003 good for the crop variety!

Reply:
oh, I just tested and this one should be fixed as well. 

Please replace `self.cropvariety = "Sugarbeet_601"`

Reply:
okay